monkey/infection_monkey/model/host.py

```diff
@@ -46,3 +46,17 @@ def __str__(self):
 
     def set_default_server(self, default_server):
         self.default_server = default_server
+
+
+def generate_victims_from_range(net_range):
```

Comment:
I'd rather you moved this method to a separate class 'VictimHostFactory' in a separate file. This will add to the readability. Once you're at it move checks from network_scanner.py into this generator method to only return victims we need to scan

Reply:
@ShayNehmad RFC?